src/stories/tasks/confirmRemove/index.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { action } from '@storybook/addon-actions';
+import { Story } from '@storybook/react/types-6-0';
+import React, { ComponentProps } from 'react';
+
+import ConfirmRemoveUserModal from '~js/components/tasks/confirmRemoveUserModal';
+import { ORG_TYPES } from '~js/utils/constants';
+
+import { withRedux } from '../../decorators';
+import { sampleUser1 } from '../../fixtures';
+
+export default {
+  title: 'Tasks/ConfirmRemove/Examples',
+  component: ConfirmRemoveUserModal,
+  decorators: [withRedux()],
+};
+
+const Template: Story<ComponentProps<typeof ConfirmRemoveUserModal>> = (
+  args,
+) => <ConfirmRemoveUserModal {...args} />;
+
+export const ConfirmRemove = Template.bind({});
+
+ConfirmRemove.args = {
+  isOpen: true,
+  waitingToRemoveUser: {
+    type: ORG_TYPES.DEV,
+    assignee: null,
+    shouldAlertAssignee: false,
+  },
+  handleClose: action('handleClose'),
+  handleCancel: action('handleCancel'),
+  handleAssignUser: action('handleAssignUser'),
+};
+ConfirmRemove.argTypes = {
+  isOpen: { control: { disable: true } },
+  waitingToRemoveUser: { control: { disable: true } },
+};
```

Comment:
I see a difference in how we are displaying the confirmDelete and this confirmRemove modal example. Either way we handle it would effect a person like me 0% but is there a reason we show a toggle for the `isOpen` control on one but not the other?  

![Screen Shot 2021-07-13 at 4 35 55 PM](https://user-images.githubusercontent.com/1581694/125528843-deb14890-3b4d-469d-9e44-28381dccd52f.jpg)